open-craft/opencraft
# Beta test registration

This pull request adds a registration form for the OpenCraft beta. The form is available at `/beta/register/`.

#### Testing

- Checkout this branch
- Run the migrations
- In a separate terminal window inside the development VM, start a dummy smtp server with `sudo python -m smtpd -n -c DebuggingServer localhost:25`
- Start the development server
- Navigate to http://localhost:5000/beta/register/
- Play around with the input fields. Error messages should appear for invalid input
- Submit the form. You should see 2 verification emails in the smtp server output
- Follow the links in the verification emails
- In the django admin, check that a registration, user, and profile have been created, and that the email addresses have been verified

Note that I have not done any styling. This will be done by a web design agency next month.
